{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "450b957e_1ece21ae",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 10,
      "author": {
        "id": 1003043
      },
      "writtenOn": "2021-02-08T13:11:19Z",
      "side": 1,
      "message": "19.3.1 (\u003c20) actually can work, even tough in the issue comments only 9.x and 21.x was tested.",
      "revId": "b251ec362c8cc16c01bc54c6e643d2de3e418fae",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "e521d97f_fb048870",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1003043
      },
      "writtenOn": "2021-02-08T12:48:30Z",
      "side": 1,
      "message": "installed pip-19.3.1  is the first working so pip\u003c20 might work.",
      "revId": "b251ec362c8cc16c01bc54c6e643d2de3e418fae",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}